player.js

```diff
@@ -0,0 +1,64 @@
+var Scrabble = require("./scrabble.js");
+
+function Player(name) {
+  this.name = name;
+  this.plays= [];
+
+};
+
+// prototype makes it an instance method
+Player.prototype.play = function(word) {
+  if(this.hasWon() !== true) {
+    this.plays.push(word)
+  } else {
+    return false
+  };
+  //returns false if player has already won
+};
+
+//total_score(): Function which sums up and returns the score of the players words
+Player.prototype.total_score = function() {
+  total = 0
+  for(var word of this.plays) {
+   total = total + Scrabble.score(word)
```

Comment:
Love how you separated out the score calculations from the total score function!